src/System.IO.Pipes/tests/packages.config

```diff
@@ -5,8 +5,10 @@
   <package id="System.IO" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
   <package id="System.Runtime" version="4.0.20-beta-22703" targetFramework="portable-net45+win" />
   <package id="System.Runtime.Handles" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Security.Principal" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
   <package id="System.Text.Encoding" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
   <package id="System.Threading.Tasks" version="4.0.10-beta-22703" targetFramework="portable-net45+win" />
+  <package id="System.Threading.Overlapped" version="4.0.0-beta-22703" targetFramework="portable-net45+win" />
```

Comment:
System.IO.Pipes.dll relies on it as part of its async I/O implementation, and without this is the tests' packages.config, the test was failing with a file not found error for the Overlapped.dll.